fluentui_core/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.microsoft.fluentui.core" />
```

Comment:
The package here says com.microsoft.fluentui.core
But in the kotlin filesour package only has a structure till com.microsoft.fluentui
Are we sure it will not cause an issue?
Also the way we refer R is com.microsoft.fluentui.core.R
@mnkshukl your thoughts?

Reply:
We have to give one package name to the module so that all common resources are accessible to other packages within the same module - import com.microsoft.fluentui.core.R, hence this is required. @mnkshukl had done this in first modular code for bottomsheet as well